dashboard-for-desktop/articles/dashboard-designer/appearance-customization/coloring/coloring-concepts.md

```diff
@@ -3,61 +3,96 @@ title: Coloring Concepts
-
-![Coloring_DimensionColorByItem](../../../../images/img25374.png)
-
-### Coloring Measures
-
-To specify the coloring mode for dashboard item measures, click the [menu button](../../ui-elements/data-items-pane.md) of any measure and use the **Color by** submenu. For instance, the image below shows the Pie dashboard item whose measures are colored by hue.
-
-![Coloring_MeasuresColorByItem](../../../../images/img25376.png)
-
-If you enabled coloring by hue for several dimensions/measures, all combinations of dimension values/measures will be automatically colored using different colors from the default palette. To learn how to customize these colors, see [Customizing a Color Scheme](customizing-a-color-scheme.md).
\ No newline at end of file
+# Coloring 
+
+The Dashboard Designer associates dimension values/measures and specified colors to paint dashboard item elements. This topic describes how to configure color settings in the WinForms Designer.
+
+## Coloring Basics
+
+The following concepts are common to both Desktop and Web Dashboard controls:
+
+* Color Modes
+* Color Schemes
+* Supported Dashboard Items
+
+Refer to the following topic for more information about common concepts: [Coloring Basics](../coloring.md).
+
+## Color Mode: None 
+
+You can disable default color variation for dashboard item elements.  
+
+If you add a [TreeMap](../../designing-dashboard-items/treemap.md) to a dashboard, individual elements (titles) use different colors: 
+
+![win-coloring-treemap-default](../../../../images/win-coloring-treemap-default.png)
+
+_State_ values (Arguments) use different colors. To disable color variation, go to the Argument settings and select **Color by | None**.
+
+![win-coloring-treemap-none](../../../../images/win-coloring-treemap-none.png)
+
+Add a [Chart](../../designing-dashboard-items/chart.md) with the same Argument and Value as in the TreeMap: 
+
+![win-coloring-treemap-chart-none](../../../../images/win-coloring-treemap-chart-none.png)
+
+**Default** means **None** for chart arguments.  
+
+## Color Mode: Hue 
+
+You can enable colors in both previously added items.
+
+Set the TreeMap's color mode to **Default** or **Hue**: 
+
+![win-coloring-chart-none-treemap-hue](../../../../images/win-coloring-chart-none-treemap-hue.png)
+
+
+In the Chart settings, move _State_ from Arguments to Series. **Default** now means **Hue** in this new context. The ![ColoringIndicator](../../../../images/coloringindicator25453.png) indicator on the data item shows that color variation by hue is enabled.
+
+![win-coloring-chart-treemap-hue](../../../../images/win-coloring-chart-treemap-hue.png)
+
+Add _Category_ as a chart argument and switch to 100% Stacked View:  
+
+![win-coloring-full-stacked-chart](../../../../images/win-coloring-full-stacked-chart.png)
+
+## Use Global Color Scheme 
+
+The same _State_ data items use identical colors. Dashboard constructs a **Global Color Scheme** for this purpose. 
```

Comment:
`The same _State_ data items use identical colors.`
Do you mean:

`Data items with the same _State_ use identical colors.`

Reply:
Thanks for the clarification. Could we use your suggestion? For example:

`State data items in the chart and treemap use the same color. The dashboard constructs a **Global Color Scheme** for this purpose.`